src/lib/style/components/_favorites-page.scss

```diff
@@ -23,5 +27,8 @@
     > td:first-child { // favorite star header
      width: 50px;
    }
+   > td:nth-child(3) svg {
+     color: #909090; // SWC-5709: entity type icon color
```

Comment:
fix entity icon color in the Favorites table (design indicates that this is temporary until a more general rule can be defined)